### DISCUSSION TITLE
part-ii/2017-4/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#61] [Question comments]
Part II 2017 Algebraic Topology: Paper 3, Section II, I 

https://questions.tripos.org/part-ii/2017-4/

**Comment by eha1234** [comment]
<details><summary>[spoilers]</summary>

$\mathbb{Z}^{nCk}$ for  $* = k, 0 \leq k \leq n, 0$ otherwise
</details>

**Reply by artie2000** [reply]
<details>
<summary>Idea for computation:</summary>

Write $S^1=I\cup I$, with $I\cap I=\{\bullet,\bullet\}$, that is, divide the circle into left and right halves, meeting at two distinct points.

Then $T_n=(T_{n-1}\times I)_1\cup(T_{n-1}\times I)_2$, with $(T_{n-1}\times I)_1\cap(T_{n-1}\times I)_2=T_{n-1}\sqcup T_{n-1}$.

Then MV (you need to explicitly compute the inclusion map).

</details>
